src/init.cpp

```diff
@@ -1853,7 +1852,8 @@ bool AppInitMain(const util::Ref& context, NodeContext& node)
         vImportFiles.push_back(strFile);
     }
 
-    threadGroup.create_thread([=, &chainman] { ThreadImport(chainman, vImportFiles); });
+    std::thread import(&TraceThread<std::function<void()>>, "loadblk", [=, &chainman]{ ThreadImport(chainman, vImportFiles); });
+    import.detach();
```

Comment:
nit: The `AppInitMain()` is a way large function. Could the scope of the `import` variable be limited by block?